src/sdk/namespace_fs.js

```diff
@@ -1575,6 +1575,14 @@ class NamespaceFS {
             const fs_context = this.prepare_fs_context(object_sdk);
             await this._load_multipart(params, fs_context);
             await this._check_path_in_bucket_boundaries(fs_context, params.mpu_path);
+            let create_multipart_upload_params = {};
+            try {
+                const { data } = await nb_native().fs.readFile(fs_context, path.join(params.mpu_path, 'create_object_upload'));
+                create_multipart_upload_params = JSON.parse(data.toString());
+            } catch (error) {
+                // Let reading/parsing fail silently
+                dbg.warn('failed to read/parse create_object_upload for mpu_path:', params.mpu_path, 'error:', error);
+            }
```

Comment:
I don't see why should we fail silently. I would keep it just like in complete_multipart_upload:
```suggestion
            const { data: create_params_buffer } = await nb_native().fs.readFile(
                fs_context,
                path.join(params.mpu_path, 'create_object_upload')
            );
```